code/game/machinery/doors/airlock.dm

```diff
@@ -1109,13 +1109,15 @@ About the new airlock wires panel:
 	if(istype(I, /obj/item/twohanded/fireaxe)) //let's make this more specific //FUCK YOU
 		var/obj/item/twohanded/fireaxe/F = I
 		if(F.wielded)
-			spawn(0)
-				if(density)
-					open(1)
+			if(density)
+				playsound(src, 'sound/machines/airlock_alien_prying.ogg', 100, 1) //is it aliens or just the CE being a dick?
+				prying_so_hard = TRUE
+				if(do_after(user, 75, target = src))
+					prying_so_hard = FALSE
+					if(density && !open(TRUE))
+						to_chat(user, "<span class='warning'>Despite your attempts, [src] refuses to open.</span>")
 				else
```

Comment:
Judging by the error message, this `else` corresponds to `if(F.wielded)` but the indentation makes it an error for failing `do_after()` (i.e. moving away after you start prying, and interrupting it)

Reply:
else was moved